### PR TITLE
support deployment into new or existing vnet

### PR DESCRIPTION
**Pull request by netwmr01**
support deployment into new or existing vnet

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@netwmr01__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: cloudera-on-centos/azuredeploy.json [hunk 1]
```diff
@@ -231,7 +249,9 @@
     "addressPrefix":"[parameters('addressPrefix')]",
     "subnetPrefix":"[parameters('subnetPrefix')]",
     "nodeAddressPrefix":"[parameters('nodeAddressPrefix')]",
-    "scriptsUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/cloudera-on-centos",
+    "scriptsUri": "https://raw.githubusercontent.com/netwmr01/azure-quickstart-templates/master/cloudera-on-centos",
```
Comment (singhkay) [inline cloudera-on-centos/azuredeploy.json]: Should point to Azure location

File: cloudera-on-centos/ds13.json [hunk 2]
```diff
@@ -228,7 +249,9 @@
     "addressPrefix":"[parameters('addressPrefix')]",
     "subnetPrefix":"[parameters('subnetPrefix')]",
     "nodeAddressPrefix":"[parameters('nodeAddressPrefix')]",
-    "scriptsUri": "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/cloudera-on-centos",
+    "scriptsUri": "https://raw.githubusercontent.com/netwmr01/azure-quickstart-templates/master/cloudera-on-centos",
```
Comment (singhkay) [inline cloudera-on-centos/ds13.json]: same comment as above

**Comment by netwmr01** [comment]
something wrong with the server?
Server Error:{
    "error": "Deployment provisioning state was not successful\nThe resource operation completed with terminal provisioning state 'Failed'.\n{\r\n  \"error\": {\r\n    \"code\": \"OperationNotAllowed\",\r\n    \"message\": \"Operation 'Update VM' is not allowed on VM 'ci6090e446-dn0' since the VM is marked for deletion.\"\r\n  }\r\n}\n{\r\n  \"error\": {\r\n    \"code\": \"OperationNotAllowed\",\r\n    \"message\": \"Operation 'Update VM' is not allowed on VM 'ci6090e446-dn2' since the VM is marked for deletion.\"\r\n  }\r\n}\n",

**Comment by singhkay** [comment]
ci passed, merged

**Pull request merged**